Sources/AppStoreConnectCLI/Commands/Certificates/ListDownloadCertificatesCommand.swift

```diff
@@ -70,19 +56,19 @@ struct ListDownloadCertificates: CommonParsableCommand {
         if let downloadPath = downloadPath {
             try certificates.forEach { (certificate: Certificate) in
                 guard let content = certificate.content else {
-                    throw CommandError.invalidContent
+                    throw CertificatesError.invalidContent
                 }
 
                 do {
-                    let folder = try Folder(path: downloadPath)
-
-                    let file = try folder.createFile(
+                    let file = try File.createFile(
+                        in: downloadPath,
                         named: "\(certificate.serialNumber ?? "serial").cer",
-                        contents: Data(base64Encoded: content)
+                        with: content
```

Comment:
I'm not sure this is actually necessary to be extracted out to a helper method.
This code is as simple when it is just:
```swift
try Folder(path: downloadPath)
    .createFile(
        named:"\(certificate.serialNumber ?? "serial").cer", 
        contents: Data(base64Encoded: content)
    )
```